backend/typescript/middlewares/validators/schedulingValidators.ts

```diff
@@ -120,8 +139,8 @@ export const updateSchedulingDtoValidator = async (
       .status(400)
       .send(getApiValidationError("volunteerNeeded", "boolean"));
   }
-  if (req.body.frequency && !validatePrimitive(req.body.frequency, "string")) {
-    return res.status(400).send(getApiValidationError("frequency", "string"));
+  if (req.body.status && !Object.values(DayPart).includes(req.body.status)) {
+    return res.status(400).send(getApiValidationError("dayPart", "string"));
```

Comment:
if you meant to check status here
```suggestion
  if (req.body.status && !Object.values(Status).includes(req.body.status)) {
    return res.status(400).send(getApiValidationError("status", "string"));
```